test/fail_compilation/fail2789.d

```diff
@@ -1,40 +1,8 @@
-/*
-https://issues.dlang.org/show_bug.cgi?id=18385
-
-TEST_OUTPUT:
----
-fail_compilation/fail2789.d(15): Error: function `fail2789.A2789.m()` conflicts with previous declaration at fail_compilation/fail2789.d(10)
----
-*/
-#line 7
-
-class A2789
-{
-    int m()
-    {
-        return 1;
-    }
-
-    float m()       // conflict
```

Comment:
I suspect the reason this is disallowed is because of inheritance

Reply:
Ah yeah right, damn. Covariance would be awkward. Reenable the check for methods?